save-core/src/commonNonJsMain/kotlin/com/saveourtool/save/core/Save.kt

```diff
@@ -147,12 +151,10 @@ class Save(
         logDebug("=> Executing plugin: ${plugin::class.simpleName} for [${plugin.testConfig.location}]")
         reporter.onPluginExecutionStart(plugin)
         try {
-            val testRepositoryRootPath = plugin.testConfig.getRootConfig().location
-
             plugin.execute()
                 .onEach { event ->
                     // calculate relative paths, because reporters don't need paths higher than root dir
-                    val resourcesRelative = event.resources.withRelativePaths(testRepositoryRootPath)
```

Comment:
previously we took it from plugin -- do you sure that it has the same value as from `saveProperties`?

Reply:
here we change logic to be compliant with the same logic from save-cloud, in save cloud it expect tests in folders which are  relative to testRootPath, when this path is not the same as test repository root path when we cant download debug info in gui